.github/workflows/triage_prs.yml

```
@@ -22,7 +22,7 @@ jobs:
             echo "should_add=true" >> $GITHUB_OUTPUT
           fi
       - name: "Add community PR to triaging project"
-        if: steps.check_author.outputs.should_add == 'true'
+        if: steps.check_author.outputs.should_add == 'true' && secrets.ADD_TO_PROJECT_PAT != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)